_data/simple-icons.json

```
@@ -5492,6 +5492,11 @@
             "hex": "246FDB",
             "source": "https://tagmanager.google.com/#/home"
         },
+        {
+            "title": "Google Tasks",
+            "hex": "2684FC",
```

Should we use the standard Google Blue here? 🤔 
```suggestion
            "hex": "4285F4",
```

---

They specifically use `#2684FC` here, so I'd say we stick with it (at least for now!)